lib/payload.json

```diff
@@ -137,6 +198,55 @@
             }
           },
           "additionalProperties": false
+        },
+        "water": {
+          "type": "object",
+          "properties": {
+            "leak": {
+              "type": "boolean",
+              "description": "Status of water leaking (true/false)"
+            }
+          },
+          "additionalProperties": false
+        },
+        "action": {
+          "type": "object",
+          "properties": {
+            "motion": {
+              "type": "object",
+              "properties": {
+                "detected": {
+                  "type": "boolean",
+                  "description": "Motion detected"
+                },
+                "count": {
+                  "type": "number",
+                  "description": "Number of motion events (count)"
+                }
+              },
+              "additionalProperties": false
+            },
+            "doorState": {
+              "type": "string",
+              "description": "State of door or window",
```

Comment:
Door and window are two separate things.

Reply:
These are usually reed/contact switches, used on doors or windows. not sure how to properly capture it all.

Reply:
Then it's a more generic open/close state, don't mention door/window then.